### DISCUSSION TITLE
How to separate voice from different people in audios with whisper?

### DISCUSSION THREAD

**Discussion by rodrigo121998** [openai/whisper#953] [Show and tell]
For example, i have audios where two people are talking. How can i separate or get the text just from one person?

**Comment by glangford** [comment]
This has been covered in many past discussions. Search for "diarization". In particular, see
* https://github.com/openai/whisper/discussions/827
* https://github.com/openai/whisper/discussions/265
* https://github.com/openai/whisper/discussions/264
